### PR TITLE
Bump vtkjs

### PR DESCRIPTION
**Pull request by bnmajor**
Bump `vtk.js` version `22.6.1` -> `25.15.1`.

- There is a bug with the `vtkCubeAxesActor` in `v26.0.0` where axes labels are not always rendered correctly or at all. Until this can be investigated further and the issue pinned down we will pin to `v25.15.1` (the last available release where the bug is not seen).
- In `v24.0.0` some breaking changes were introduced that seem to affect the way events are captured and passed along. This PR adds new event handlers to catch double-click to zoom out as well as right-click for the context menu.
- We are now using scientific notation for the scalar bars so that text does not overlap the plots.

### CODE REVIEW

File: client/src/components/widgets/RenderWindow/script.js [hunk 1]
```diff
@@ -41,12 +41,31 @@ export default {
...
+          if (plotCell.$refs[`${row}-${col}`]?.renderer) {
+            plotCell.$refs[`${row}-${col}`].resetZoom();
           }
         });
       }
     },
+    resetZoom(e) {
+      if (!this.syncZoom && !this.selectTimeStep) {
+        document.elementsFromPoint(e.clientX, e.clientY).forEach((elem) => {
+          console.log(elem.classList);
```
Comment (jourdain) [inline client/src/components/widgets/RenderWindow/script.js:54]: console.log

File: client/src/components/widgets/RenderWindow/script.js [hunk 2]
```diff
@@ -41,12 +41,31 @@ export default {
...
+          if (elem.classList[0] === "plot") {
+            console.log(elem.__vue__);
```
Comment (jourdain) [inline client/src/components/widgets/RenderWindow/script.js:56]: console.log

File: client/src/utils/vtkCustomCubeAxesActor.js [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+import vtkCubeAxesActor from "@kitware/vtk.js/Rendering/Core/CubeAxesActor";
+import * as d3 from "d3-scale";
+
+const facesToDraw = [false, false, false, false, false, true];
+const edgesToDraw = [0, 0, 0, 0, 1, 2, 2, 1, 0, 0, 0, 0];
+
+function vtkCustomCubeAxesActor(publicAPI, model) {
+  model.classHierarchy.push("vtkCustomCubeAxesActor");
+  model.cubeAxesActorHelper = vtkCubeAxesActor.newCubeAxesActorHelper();
+  global.cubeModel = model;
```
Comment (jourdain) [inline client/src/utils/vtkCustomCubeAxesActor.js:11]: why global?

File: client/src/utils/vtkCustomCubeAxesActor.js [hunk 4]
```diff
@@ -0,0 +1,83 @@
+import macro from "@kitware/vtk.js/macros";
+import vtkCubeAxesActor from "@kitware/vtk.js/Rendering/Core/CubeAxesActor";
+import * as d3 from "d3-scale";
+
+const facesToDraw = [false, false, false, false, false, true];
+const edgesToDraw = [0, 0, 0, 0, 1, 2, 2, 1, 0, 0, 0, 0];
+
+function vtkCustomCubeAxesActor(publicAPI, model) {
+  model.classHierarchy.push("vtkCustomCubeAxesActor");
+  model.cubeAxesActorHelper = vtkCubeAxesActor.newCubeAxesActorHelper();
```
Comment (jourdain) [inline client/src/utils/vtkCustomCubeAxesActor.js:10]: What do you use that helper for?

File: client/src/components/widgets/RenderWindow/script.js (hunk 1 above)
Comment (bnmajor) [reply client/src/components/widgets/RenderWindow/script.js]: Removed

File: client/src/components/widgets/RenderWindow/script.js (hunk 2 above)
Comment (bnmajor) [reply client/src/components/widgets/RenderWindow/script.js]: Removed

File: client/src/utils/vtkCustomCubeAxesActor.js (hunk 3 above)
Comment (bnmajor) [reply client/src/utils/vtkCustomCubeAxesActor.js]: Started from an example, just forgot to do some final cleanup.

File: client/src/utils/vtkCustomCubeAxesActor.js (hunk 4 above)
Comment (bnmajor) [reply client/src/utils/vtkCustomCubeAxesActor.js]: Nothing, also needed cleaned up.

**Comment by bnmajor** [comment]
Thanks @jourdain!

**Pull request merged**